sources/assets/Stride.Core.Assets.CompilerApp/PackageBuilder.cs

```diff
@@ -22,7 +22,7 @@
 using Stride;
 using Stride.Assets;
 using Stride.Graphics;
-using Stride.Core.VisualStudio;
+using Stride.Core.CodeEditor.VisualStudio;
```

Comment:
Unsure about the namespace `CodeEditor`. We will likely have an integrated code editor in the new studio. But here it's about supporting third parties. Maybe `CodeEditorSupport` or `CodeEditing`, or something else (`CodeEditors`)?

Reply:
Yeah, I also agree that it can be better expressed, naming sometimes is just hard😅
After some longer thought, I think it'd be valid to separate into two namespaces:
1. Stride.Core.Solution(s) - Contains classes around Solution entity and Solution management
2. Stride.Core.CodeEditorSupport - Contains classes related to Code Editor support 

Let me know what you think about this change